### DISCUSSION TITLE
How to examine APIs in network tab

### DISCUSSION THREAD

**Discussion by mhabbasi-gh** [nuxt/nuxt#8964] [Questions (Nuxt 2)]
The APIs in `fetch()` hook don't appear on my network tab. Is there a way to see them on my browser's network tab?

**Comment by hazer-hazer** [comment]
Please, give us your component/page code snippet

**Comment by danielroe** [comment]
It's probably because you're making a hard reload and they are happening on the server, before it comes to your browser. If you want to force it happening in browser, you can set `fetchOnServer: true` in that component.

**Reply by TheAlexLichter** [reply]
* `fetchOnServer: false`

**Reply by danielroe** [reply]
Indeed!
